apps/alarm/package.json

```diff
@@ -26,7 +26,8 @@
       ]
     ],
     "notifications": [
-      "on-ready"
+      "on-ready",
```

Comment:
@legendecas At the alarm's case, do we need `on-ready` yet?

Reply:
`on-ready` is after wifi connected. `on-boot` is before wifi connect. 
But there is a new question, alarm will be initialized twice when device restarted. I test that it dose not affect the function.

Reply:
Sync should be performed once network is connected. Thus it would be required IMO.

Reply:
Ok, let's resolve it for now :)